### ISSUE TITLE
Grouped events with equal metadata: one card with mild seperations like 

### ISSUE THREAD

**Issue by amitay7** [daat-hamakom/maps#12]
like -------- dashed line between events

**Comment by mushon** [comment]
@amitay7 please prioritize this one

**Comment by amitay7** [comment]
@josefspr can you please enter such an event (events with equal metadata) so we can get better sense of how it should look like?

**Comment by josefspr** [comment]
what do you mean equal meta data ? I already have events that have the same description and media
attached to them, do you also need them to have the same person field ?

**Comment by amitay7** [comment]
![uploads/bf057f99-62e7-4d47-b2c4-6f0e1948d0a6/blob](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2Fbf057f99-62e7-4d47-b2c4-6f0e1948d0a6%2Fblob)

**Comment by josefspr** [comment]
Not sure I understand the example. here are 2 events with the same data:
https://daat-hamakom-data.herokuapp.com/admin/maps/event/141/change/?_changelist_filters=project__id__exact%3D4
https://daat-hamakom-data.herokuapp.com/admin/maps/event/142/change/?_changelist_filters=project__id__exact%3D4

**Comment by mushon** [comment]
From the example we see here, the design doesn't really answer the need of these similar events. It looks like the only difference between them is their location. @josefspr is that true of all of these cases?

**Comment by josefspr** [comment]
yes, they contain exact data/media only differ in place and time

**Comment by mushon** [comment]
ok, how about [this kind of behavior](https://invis.io/HC6EQ6L7Q#/178587825_Selected_Item-Multiple_Locations) - here shown applied to different locations but could also work with multiple times and with the grouping of the two (multiple times and locations) in this format:

```
Odessa, Ukraine / Jan 1913 - Aug 1914;
Berlin, Germany / Feb 1916 - Mar 1916;
Warsaw, Poland / Jul 1917 - Jan 1918;
+ 16 more
```

Optionally we could also choose to go with the above format for every multiplication like this:

```
Odessa, Ukraine / Jan 1913 - Aug 1914;
Odessa, Ukraine / Feb 1916 - Mar 1916;
Odessa, Ukraine / Jul 1917 - Jan 1918;
+ 16 more
```

Or this:

```
Odessa, Ukraine / Jan 1913 - Aug 1914;
Berlin, Germany / Jan 1913 - Aug 1914;
Warsaw, Poland / Jan 1913 - Aug 1914;
+ 16 more
```

What say you?

**Comment by josefspr** [comment]
first of all, how does the map know the events are identical? - will there be something in the admin
or automatically (same title, data etc) ? 
How do these events behave, they light up together on hovering, are they a group ?
Any case should keep it simple implementation.
I didn't exactly understand the difference between the 3 versions. In most cases the place is different
and perhaps the time so
place / time
place / time 
...
seems right

**Comment by mushon** [comment]
@yuvadm please respond to the admin question above
As for whether they would be a group, I find it hard to think of a reason why such events will not be a group.

**Comment by yuvadm** [comment]
The only reasonable  behavior for this seems to be to manually include them all in a group annotation. We don't have any other logic for grouping "similar" events at any level.

**Comment by josefspr** [comment]
In most cases I already defined most of these cases as groups, so no special implementation or design is necessary (except completing implementation of the group card). We can leave it as it is, since this type 
of event group is not so common.

**Comment by mushon** [comment]
ok, so I'm reassigning this to @yuvadm for implementation
